unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Rendering/RenderingController.cs

```diff
@@ -14,7 +14,7 @@ public class RenderingController : MonoBehaviour
     public CompositeLock renderingActivatedAckLock = new ();
 
     private bool activatedRenderingBefore { get; set; }
-    private bool isDecoupledLoadingScreenEnabled => DataStore.i.featureFlags.flags.Get().IsFeatureEnabled(DataStore.i.featureFlags.DECOUPLED_LOADING_SCREEN_FF);
+    private bool isDecoupledLoadingScreenEnabled => true;
```

Comment:
I think we need to remove it completely

Reply:
Agreed, but I planned on creating a new ticket to refactor RenderingController. I want to remove the ActivateRendering logic from kernel as well, and thought it would be too much to do it in this PR